### PR TITLE
Adds show_iPad option that overrides show_iOS option for banner

### PR DESCRIPTION
**Pull request by scotthasbrouck**

### CODE REVIEW

**Comment by scotthasbrouck** [comment]
@dmitrig01 ready for review

**Comment by scotthasbrouck** [comment]
yo @derrickstaten here's this

**Comment by derrickstaten** [comment]
Awesome! For me to test this, I need to pull down and recompile, pointing at localhost, right? Or can I test with a live app?

**Comment by scotthasbrouck** [comment]
You shouldn't need to recompile it, but just pull it down and open up example.html, and play around with the options object argument in the branch.banner call

File: testbeds/web/example.html [hunk 1]
```diff
@@ -217,9 +217,10 @@
         description: 'Branch Demo app!',
         openAppButtonText: 'Open',
         downloadAppButtonText: 'Download',
-        iframe: true,     // Show banner in an iframe if CSS on your page is conflicting
-        showMobile: true, // true by default, just set here for an example
-        showDesktop: true // true by default, just set here for an example
+        iframe: true,       // Show banner in an iframe if CSS on your page is conflicting
+        showiOS: true,      // true by default, just set here for an example
+        showDesktop: true,  // true by default, just set here for an example
+        showiPad: true      // true by default, just set here for an example
```
Comment (scotthasbrouck) [inline testbeds/web/example.html]: Yeah, play around with these here. I made a four by four grid of every combination of showiPad and show iOS. Then tested each of those in iPad, and iPhone, for a total of 16 tests.

Comment (scotthasbrouck) [inline testbeds/web/example.html]: Like This:

**Comment by derrickstaten** [comment]
great! I'll try to do this tonight. Getting off the train. It may be this weekend. You're a boss for cranking this out.

**Comment by scotthasbrouck** [comment]
np!

**Comment by dmitrig01** [comment]
is this waiting on @derrickstaten to review?

**Comment by scotthasbrouck** [comment]
I think so, just want to make sure it's inline with what the partner wanted

**Comment by derrickstaten** [comment]
I think that's right--showiPad should override showiOS on devices that are iPads

**Comment by derrickstaten** [comment]
:+1:

**Comment by scotthasbrouck** [comment]
Perfect, @dmitrig01 let me know if I'm good to merge and release this

**Comment by dmitrig01** [comment]
:+1:

**Comment by scotthasbrouck** [comment]
cool, I'll resolve the conflict and update changelog

**Pull request merged**